### DISCUSSION TITLE
Value for Total on graphs

### DISCUSSION THREAD

**Discussion by statisticamd** [open-sdg/open-sdg#2131] [Q&A]
Hello!
How can we change the value for Total on graphs? It is by default the value for the geographical coverage, but we would like to change it with Total. The geographical coverage is shown below the graph.

![image](https://github.com/user-attachments/assets/e6385900-02bb-4df9-bb65-707b2a5d3f89)

Thank you!

**Comment by christoph-LSN** [comment]
@statisticamd could you poste links to your indicator site, data and site repo?

**Reply by statisticamd** [reply]
The platform https://statisticamd.github.io/sdg-site-moldova/
Data repository https://github.com/statisticamd/sdg-data-moldova
Site repository https://github.com/statisticamd/sdg-site-moldova

**Comment by sarah-beasley** [comment]
Apologies for the delay on this. All three of the underlined items are controlled by the same thing - the national_geographical_coverage setting in your indicator configuration files. Therefore, whatever you write in this field, will appear for all three. If you leave it blank, then all three items will use your country name (which is Moldova as described in your site_config file in your site repository. If you want it to appear shorter on the graph, then I'd recommend leaving the field blank so it says Moldova. And if you would still like the longer explanation of the geographical coverage, then you can add this in as a [footer field feature](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#footer_fields) so it will appear in the footnotes below the graph. You could also add this information into the metadata too. I hope this helps.
